src/ai/backend/client/cli/scp.py

```diff
@@ -9,19 +9,19 @@
 @click.argument("dest",  type=str, default="work@localhost", metavar='DEST')
 @click.option('-p', '--port',  type=str, metavar='PORT', default=9922,
               help="the port number is for localhost")
-@click.option('-r',  metavar="RECURSIVE", default=False, is_flag=True,
-              help="recursive option to process directories")
-def scp(sess_name, src, dest, port, r):
+@click.option('-r',  '--recursive', default=False, is_flag=True,
+              help="recursive flag option to process directories")
+def scp(sess_name, src, dest, port, recursive):
     """SCP client to Backend.AI SSH server.
```

Comment:
Let's make this sentence to be imperative, .e.g., "Execute the scp command against the target compute session."